dsl/cleanup_apps.rb

```diff
@@ -0,0 +1,45 @@
+require "rubygems"
+require "ruby-jmeter"
+require "cgi"
+
+if ARGV.size != 3
+  puts "\n\n------- Tsuru perf test: ERROR! --------"
+  puts "      Usage:"
+  puts "      Please specify a tsuru target domain,  "
+  puts "  a thread count and a loop count for each thread.\n"
+  puts "      Example:"
+  puts "          \n\n"
+  exit 1
+end
+
+test do
+  counter "CounterConfig.name" => "app_id",
+          "CounterConfig.start" => 1,
+          "CounterConfig.incr" => 1,
+          "CounterConfig.per_user" => "false",
+          "CounterConfig.reset_on_tg_iteration" => "false"
+
+  defaults domain: ARGV[0], protocol: "https"
+
+  header [
+    { name: "Content-Type", value: "application/json" }
+  ]
+  auth_url = "/users/" + CGI.escape("administrator@gds.tsuru.gov") + "/tokens"
+  threads count: ARGV[1].to_i, loops: ARGV[2].to_i do
+    Once do
+      post url: auth_url,
+        raw_body: '{ "password": "admin123" }' do
```

Comment:
We shouldn't store the username and password in plaintext. Even if they aren't secure at the moment, they will be at some point in the near future and this will break. Is it feasible to extract them from the vault, like we [do for the end-to-end tests](https://github.com/alphagov/tsuru-ansible/blob/463fae35f6862d15d331185bd957a3c42502354c/Makefile#L72-L80)?